src/rnp/fficli.cpp

```diff
@@ -198,6 +198,37 @@ ptimestr(char *dest, size_t size, time_t t)
     return dest;
 }
 
+static bool
+cli_rnp_get_confirmation(const cli_rnp_t *rnp, const char *msg, ...)
+{
+    char    reply[10];
+    va_list ap;
+
+    while (true) {
+        va_start(ap, msg);
+        vfprintf(rnp->userio_out, msg, ap);
+        va_end(ap);
+        fprintf(rnp->userio_out, " (y/N) ");
+        fflush(rnp->userio_out);
+
+        if (fgets(reply, sizeof(reply), rnp->userio_in) == NULL) {
+            return false;
+        }
+        if (strlen(reply) > 0) {
```

Comment:
y/N means N is the default choice.
However, when I press <Enter>, the choice is not recognized.
![image](https://user-images.githubusercontent.com/23335944/104848760-66375200-58f7-11eb-8752-4aa897cc18fc.png)

Reply:
@rrrooommmaaa fixed